pkg/resource/deploy/step.go

```diff
@@ -208,6 +208,8 @@ func (s *CreateStep) Apply(preview bool) (resource.Status, StepCompleteFunc, err
 			s.new.ID = id
 			s.new.Outputs = outs
 		}
+	} else {
+		s.new.Outputs = s.new.Inputs
 	}
```

Comment:
could we flip the 'if' we're in.  i'm a big fan of `if case simpleStuff else really complex stuff` versus `if !case complex stuff else really simple stuff`.